.phpcs.xml.dist

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	name="Duplicate Post"
+	xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
+
+	<description>Duplicate Post rules for PHP_CodeSniffer</description>
+
+	<!--
+	#############################################################################
+	COMMAND LINE ARGUMENTS
+	https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml
+	#############################################################################
+	-->
+
+	<file>.</file>
+
+	<!-- Always exclude all files in version management related directories. -->
+	<exclude-pattern>*/.git/*</exclude-pattern>
+	<exclude-pattern>*/.wordpress-svn/*</exclude-pattern>
+
+	<!-- Exclude the build tool and dependency related directories. -->
+	<exclude-pattern>/vendor/*</exclude-pattern>
+	<exclude-pattern>/node_modules/*</exclude-pattern>
+	<exclude-pattern>/config/grunt/*</exclude-pattern>
```

Comment:
Question: there's also a `/config/webpack/` directory. Should that be excluded as well?

Reply:
@afercia Good question.

Actually, I think that line got brought back when I was rebasing at some point It should be removed.

JS files are no longer scanned because of limiting the ruleset to PHP extensions, so we don't actually have to exclude these directories, as they would be automatically skipped as they only contain JS/Yaml files. That applies to both the `config/grunt` as well as the `config/webpack` directories.

I've fixed this up in the "improve ruleset" commit.